package.json

```diff
@@ -17,8 +17,7 @@
     "npm": ">=7.0"
   },
   "volta": {
-    "node": "16.13.0",
-    "npm": "8.3.1"
+    "node": "22.14.0"
```

Comment:
I think we still want `npm` here, (which is now `11.1.0`)

Reply:
thanks, updated

(got here from volta telling me to run `volta pin npm@bundled` which apparently removes the npm version)

Reply:
hmm, node 18 isn't compatible with npm 11